ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -924,6 +924,33 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-private-shared-vpc-phz-sts-p1-f14
+  cron: 26 5 7,22 * *
+  remote_api: true
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and not @fips'
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
If this job is a techpreview feature, need to add "techpreview" keyword in the job name.

Reply:
Open question: do we consider using `-tp` instead of `-techpreview` in the future? b/c it's more shorter.

Reply:
yeah, you can use `tp` instead of `techpreview`, in someday, we can keep all the job consistent.